.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: LTE integ test containerized AGW
+
+on:
+  workflow_dispatch: null
+  workflow_run:
+    workflows:
+      - build-all
+    branches:
```

Comment:
`'v1.*'` branches missing

Reply:
While it might make sense to run it for release branches *in future*, currently, I cannot see any sense in it, as it will most likely not work without substantial amount of backporting on v1.8.
Additionally, I see this clearly out of scope of the ticket, such that it will make no sense at all to add it to this PR. A new story could be created if this is desired in future.

Reply:
On second thought, it does not make sense at all to add the versioned branches here, as the workflow is not designed to work with them anyway.
To generalise the workflow and make it work with any branch is also clearly out of scope of the ticket.

Reply:
It was one of the few differences between this and the other version of the PR in #14023. Just trying to avoid things getting lost.

But I see that it probably makes little sense while we always pick the latest image version. Added it to #14052.